src/raft/raft.rs

```diff
@@ -1711,6 +1766,20 @@ impl<T: Storage> Raft<T> {
             return Some(false);
         }
 
+        if self.initialized && !self.is_learner {
```

Comment:
We don't behave the same as Etcd now. I think it's reasonable but not sure if it's a good way. Another way is to add an additional flag to the messages sent to learners. Any ideas? For the reason of the changes see discussion at https://github.com/pingcap/tikv/pull/2491#discussion_r156273925.

/cc @siddontang @xiang90